tests/screenshot/apps/components/VideoPlayer.js

```diff
@@ -0,0 +1,74 @@
+import VideoPlayer from '../../../../VideoPlayer';
+import {MediaControls} from '../../../../MediaPlayer';
+
+const posterUrl = 'http://media.w3.org/2010/05/sintel/poster.png';
+const videoUrl = 'http://media.w3.org/2010/05/sintel/trailer.mp4';
```

Comment:
same above.

Reply:
In addition, Let's change media source to 'https://media.w3.org/2010/05/video/movie_90.mp4'.